sdk/avs/Azure.ResourceManager.Avs/src/Customized/WorkloadNetworkName.cs

```diff
@@ -13,24 +13,31 @@ namespace Azure.ResourceManager.Avs.Models
     /// <summary> The WorkloadNetworkName. </summary>
     public readonly partial struct WorkloadNetworkName : IEquatable<WorkloadNetworkName>
     {
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
we do not need to add this attribute to those non-public members, they are not browsable in editor already.